src/site/stages/build/page-builder/helpers.js

```diff
@@ -91,10 +88,10 @@ module.exports = {
    *
    * @return {Object} - The contents of the file.
    */
-  readEntity(type, uuid) {
+  readEntity(entityName, targetID = 'index') {
     return JSON.parse(
       fs
-        .readFileSync(path.join(contentDir, `${type}.${uuid}.json`))
+        .readFileSync(path.join(entities, `${entityName}/${targetID}.json`))
```

Comment:
So we have two places that this function needs to read from:
- From the test directory for use in the unit tests
- From the directory where we've un`tar`red the CMS export for the "real" content build

How can we can do both?

Reply:
Good question, one sec and I will update it.